var/spack/repos/builtin/packages/mgis/package.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Mgis(CMakePackage):
+    """
+    The MFrontGenericInterfaceSupport project (MGIS) provides helper
+    functions for various solvers to interact with behaviour written
+    using MFront generic interface.
+
+    MGIS is written in C++.
+    Bindings are provided for C and fortran (2003).
+    A FEniCS binding is also available.
+    """
+
+    homepage = "https://thelfer.github.io/mgis/web/index.html"
+    url      = "https://github.com/thelfer/MFrontGenericInterfaceSupport/archive/MFrontGenericInterfaceSupport-1.0.tar.gz"
+    git      = "https://github.com/thelfer/MFrontGenericInterfaceSupport.git"
+    maintainers = ['thelfer']
+
+    # development branches
+    version("master", branch="master")
+    version("rliv-1.0", branch="rliv-1.0")
+
+    # released version
+    version('1.0', sha256='279c98da00fa6855edf29c2b8f8bad6e7732298dc62ef67d028d6bbeaac043b3')
+
+    # variants
+
+    variant('c_bindings', default=True,
+            description='Enables c bindings')
+    variant('fortran_bindings', default=True,
+            description='Enables fortran bindings')
+    variant('python_bindings', default=True,
+            description='Enables python bindings')
```

Comment:
I would be fine with dropping `bindings` from all of these variant names. It's usually implied. For example, most packages that add optional Python bindings just have a `+python` variant.